### ISSUE TITLE
INTEGER + PRIMARY_KEY + AUTOINCREMENT error

### ISSUE THREAD

**Issue by shenbibo** [Kotlin/anko#469]
This happen in 0.10.1, and in the 0.10.0 is OK
when use INTEGER + PRIMARY_KEY + AUTOINCREMENT  for DB id, this will throw a exception:
android.database.sqlite.SQLiteException: near "org": syntax error (code 1): , while compiling: CREATE TABLE IF NOT EXISTS `DayWeather`(id INTEGER PRIMARY KEY org.jetbrains.anko.db.SqlTypeModifierImpl@1a12540, cityId TEXT

when i see the source code in sqlTypes.kt, i found this error (SqlTypeImpl.plus):
```kotlin
    override fun plus(m: SqlTypeModifier): SqlType {
        return SqlTypeImpl(name, if (modifiers == null) m.modifier else "$modifiers $m")
    }
```koltin
the end "$m" should change to "${m.modifiers}"

**Comment by guidev224** [comment]
The error is not due to the AUTOINCREMENT. It occurs when you concatenate more than two values. I stuck on the same issue.

**Comment by monxalo** [comment]
#374 already resolved this. Strangely it wasn't merged into the 0.10 branch.

**Comment by 4u7** [comment]
Fixed with a9fb4771c4ac8a211839bdb9174bf967595a36aa
